src/shared/libs/iota/utils.js

```diff
@@ -191,6 +191,36 @@ export const isValidAddress = (address) => {
     return false;
 };
 
+/**
+ * Checks if the last trit is 0
+ *
+ * @method isLastTritZero
+ * @param {string} address
+ *
+ * @returns {boolean}
+ */
+export const isLastTritZero = (address) => {
+    if (!/[E-V]/.test(address.slice(80, 81))) {
```

Comment:
Just a suggestion but if you're not doing any validation on the parameter perhaps this can be simplified to just `export const isLastTritZero = (address) => !/[E-V]/.test(address.slice(80, 81))` i.e., no need for explicit returns.